docs/UsersGuide/source/ConfigParameters.inc

```diff
@@ -233,106 +668,347 @@ Set parameters associated with the fixed (i.e. static) files.  For the main NOAA
+
+``SPPT_SFCLIMIT``: (Default: "TRUE")
+   When "TRUE", tapers the SPPT perturbations to zero at the model’s lowest level, which helps reduce model crashes. 
+
+``USE_ZMTNBLCK``: (Default: "FALSE")
+   When "TRUE", do not apply perturbations below the dividing streamline that is diagnosed by the gravity wave drag, mountain blocking scheme
+
+Stochastic Kinetic Energy Backscatter (SKEB) Parameters
+----------------------------------------------------------
+
+``DO_SKEB``: (Default: "FALSE")
+   Flag to turn Stochastic Kinetic Energy Backscatter (SKEB) on or off. SKEB adds wind perturbations to the model state. Perturbations are random in space/time, but amplitude is determined by a smoothed dissipation estimate provided by the :term:`dynamical core`. SKEB addresses errors in the dynamics more active in the mid-latitudes.
+
+``ISEED_SKEB``: (Default: "3")
+   Seed for setting the SHUM random number sequence.
+
+``SKEB_MAG``: (Default: "0.5") 
+   Amplitude of random patterns. Corresponds to the variable ``skeb`` in ``input.nml``.
+
+``SKEB_LSCALE``: (Default: "150000")
+   Decorrelation spatial scale in meters. 
+
+``SKEB_TSCALE``: (Default: "21600")
+   Decorrelation timescale in seconds. Corresponds to the variable ``skeb_tau`` in ``input.nml``.
+
+``SKEB_INT``: (Default: "3600")
+   Interval in seconds to update random pattern (optional). Perturbations still get applied every time-step. Corresponds to the variable ``skebint`` in ``input.nml``.
+
+``SKEBNORM``: (Default: "1")
+   Patterns:
+      * 0-random pattern is stream function
+      * 1-pattern is K.E. norm
+      * 2-pattern is vorticity
+
+..
+   COMMENT: Stochastic physics docs say the default is zero... Have the numbers changed? Or did the default change?
+
+
+``SKEB_VDOF``: (Default: "10")
+   The number of degrees of freedom in the vertical for the SKEB random pattern. 
+
+..
+   COMMENT: The vertical what?
+
+.. _SPP:
+
+SPP Stochastic Physics Parameters
+------------------------------------
+
+Set default Stochastically Perturbed Parameterizations (SPP) stochastic physics options. Unlike `SPPT physics <SPPT>`, SPP is applied as part of the physics run, not post. Each SPP option is an array, applicable (in order) to the scheme/parameter listed in ``SPP_VAR_LIST``. Enter each value of the array in ``config.sh`` as shown below without commas or single quotes (e.g., ``SPP_VAR_LIST=( "pbl" "sfc" "mp" "rad" "gwd"`` ). Both commas and single quotes will be added by Jinja when creating the namelist.
+
+.. note::
+   SPP is currently only available for specific physics schemes used in the RAP/HRRR physics suite. Users need to be aware of which :term:`SDF` is chosen when turning this option on. 
+
+..
+   COMMENT: Is this still the case? 
+   COMMENT: What is the difference between SPPT (above) and SPP?
+
+``DO_SPP``: (Default: "false")
+   Flag to turn SPP on or off. 
+
+..
+   COMMENT: What does SPP do? Need a description similar to the SKEB/SHUM/SPPT descriptions. 
```

Comment:
I hope @JeffBeck-NOAA can help here

Reply:
SPP perturbs parameters and other fields with unknown or uncertain magnitudes within the physics code based on ranges provided by physics experts.